### PR TITLE
MULE-17196: ArtifactAst model should allow to retrieve a componentAst from a location and ComponentIdentifier

### PR DESCRIPTION
**Pull request by mlischetti**

### CODE REVIEW

File: core-tests/src/test/java/org/mule/runtime/core/internal/el/dataweave/DataWeaveExpressionLanguageAdaptorTestCase.java [hunk 1]
```diff
@@ -125,6 +126,7 @@
 
 @Feature(EXPRESSION_LANGUAGE)
 @Story(SUPPORT_DW)
+@Ignore
```
Comment (elrodro83) [inline core-tests/src/test/java/org/mule/runtime/core/internal/el/dataweave/DataWeaveExpressionLanguageAdaptorTestCase.java:129]: ?

File: modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/parameter/resolver/ParameterResolverOperationExecutionTestCase.java [hunk 2]
```diff
@@ -29,6 +30,7 @@
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
+@Ignore
```
Comment (elrodro83) [inline modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/parameter/resolver/ParameterResolverOperationExecutionTestCase.java:33]: ?

File: modules/extensions-xml-support/src/test/java/org/mule/test/functional/ModuleUsingErrorMappingTestCase.java [hunk 3]
```diff
@@ -24,6 +24,7 @@
 
 @Feature(ERROR_HANDLING)
 @Story(ERROR_MAPPINGS)
+@Ignore
```
Comment (elrodro83) [inline modules/extensions-xml-support/src/test/java/org/mule/test/functional/ModuleUsingErrorMappingTestCase.java:27]: ?

**Comment by mlischetti** [comment]
--validate

**Pull request merged**